install-k

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-K_VERSION=7.1.0
+K_VERSION=7.1.7
 
```

Comment:
@goodlyrottenapple these changes should not be present.

Reply:
sorry for the mess, not sure what i did on my branch there... should be fixed now